salt/salt.spec

```
@@ -54,8 +54,11 @@
 %if %{without systemd}
 %define service_del_preun echo %{*}
 %endif
-
+%if 0%{?suse_version} == 1500 && 0%{?sle_version} >= 150700
```

Looks like we can remove the first part of the condition here with no affecting the result:
```suggestion
%if 0%{?sle_version} >= 150700
```

---

Please check what should happen with suse_version = 1600 and if sle_version is set everywhere. 
AFAIK in OBS some projects do not have it. Tumbleweed?

---

Oh, yes it's true, TW has `%sles_version 0`, but `%suse_version 1699`. Maybe this way:
```suggestion
%if 0%{?suse_version} >= 1600 || 0%{?sle_version} >= 150700
```

---

Hmmm, I only want this to apply to SLE15SP7, as the previous `%{?sle15allpythons}` macro makes SP7 to produce a `python3-salt` package. For TW and SLE16, we should be fine already, as the `%{?sle15allpythons} macro is not producing `python3-salt` package.